rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/TimeSlotPolicy.java

```diff
@@ -57,62 +57,37 @@
 @ActiveObject
 public class TimeSlotPolicy extends NodeSourcePolicy implements InitActive {
 
-    /**
-     * Timer task acquired all node from infrastructure
-     */
-    private class AcquireTask extends TimerTask {
-        @Override
-        public void run() {
-            synchronized (timer) {
-                acquireAllNodes();
-            }
-        }
-    }
+    private static final String TIMER_NAME = "TimeSlotPolicy Timer";
 
-    /**
-     * Timer task released all node from infrastructure
-     */
-    private class ReleaseTask extends TimerTask {
-        @Override
-        public void run() {
-            synchronized (timer) {
-                // security trick
-                thisStub.removeAllNodes(preemptive);
-            }
-        }
-    }
-
-    /** Date formatter */
-    public static transient DateFormat dateFormat = SimpleDateFormat.getDateTimeInstance(SimpleDateFormat.SHORT,
-                                                                                         SimpleDateFormat.FULL,
-                                                                                         new Locale("en"));
+    public static final transient DateFormat DATE_FORMAT = SimpleDateFormat.getDateTimeInstance(SimpleDateFormat.SHORT,
+                                                                                                SimpleDateFormat.FULL,
+                                                                                                new Locale("en"));
```

Comment:
The dates that are displayed should be YYYY/MM/DD, 24H format

Reply:
Ok, thanks, I have added this remark in the open issue https://github.com/ow2-proactive/scheduling/issues/3197.